### DISCUSSION TITLE
能够增加单纯执行sql的方法，类似于EF Core: db.Database.ExecuteSqlCommand

### DISCUSSION THREAD

**Discussion by MaxNie87** [dotnetcore/FreeSql#885] [General]
执行数据库的版本升级脚本

**Comment by luoyunchong** [comment]
```
fsql.Ado.ExecuteNonQuery(cmdText, new{});
```
https://github.com/dotnetcore/FreeSql/wiki/ADO
https://github.com/dotnetcore/FreeSql/blob/master/FreeSql/Interface/IAdo.cs#L183
